### PR TITLE
raw_input() was removed in Python 3

### PR DESCRIPTION
**Pull request by cclauss**
__raw_input()__ was removed in Python 3 in favor of __input()__.  We __import six__ so that [__six.moves.input()__](https://pythonhosted.org/six/#module-six.moves) works identically in both Python 2 and Python 3.  We also add .strip() to gracefully deal with leading or trailing whitespace in the user input as well as lower() to gracefully deal with capital as well as lowercase letters.

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by rmlarsen** [comment]
@fchollet would you please review this? Thanks.

File: tensorflow/python/keras/_impl/keras/utils/io_utils.py [hunk 1]
```diff
@@ -19,9 +19,9 @@
 from __future__ import print_function
 
 from collections import defaultdict
-import sys
 
 import numpy as np
+from six.moves import input
```
Comment (fchollet) [inline tensorflow/python/keras/_impl/keras/utils/io_utils.py]: Prefer `import six` then `six.moves.input`

**Comment by cclauss** [comment]
Modified to be in sync with upstream:
* https://github.com/keras-team/keras/blob/master/keras/utils/io_utils.py#L144-L148

**Comment by tensorflowbutler** [comment]
Nagging Assignee @rmlarsen: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by rmlarsen** [comment]
@fchollet could you take another look and approve, if possible?

**Review by rmlarsen — APPROVED** [review]
